pydoctor/test/epydoc/test_restructuredtext.py

```diff
@@ -26,3 +29,80 @@ def test_rst_anon_link_email() -> None:
     assert html.startswith('<a ')
     assert ' href="mailto:postmaster@example.net"' in html
     assert html.endswith('>mailto:postmaster@example.net</a>')
+
+# THIS IS HELPER METHODS
```

Comment:
The fact that their name doesn't start with `test_` already identifies them as helper methods, so this comment is not needed.